src/components/VulnerabilitiesBlock/VulnerabilitiesBlockInner.tsx

```diff
@@ -43,6 +50,22 @@ const VulnerabilitiesBlockInner: React.FC<{
           {region.shortName} {regionDescription}
         </SummaryText>
         <SummaryText>{communityVulnerabilityQuote}</SummaryText>
+        {surgoUrl && (
+          <SummaryText>
+            <TextButton
+              href={surgoUrl}
+              onClick={() =>
```

Comment:
since we're using TextButton, which is part of the button system, the props are a little different. [see here for one thats already implemented](https://github.com/covid-projections/covid-projections/blob/develop/src/screens/DataApi/DataApi.tsx#L49). we dont need to pass an onClick with trackEvent, because that is already implemented/called [in the BaseButton that is at the root of TextButton](https://github.com/covid-projections/covid-projections/blob/develop/src/components/ButtonSystem/BaseButton/BaseButton.tsx#L35) (/at the root of all buttons in the button system)

Reply:
so in this case we just need: `href`, `trackingCategory`, and `trackingLabel`